src/static/javascripts/components/dashboard.jsx

```diff
@@ -136,47 +152,50 @@ var MainDiv = React.createClass({
 });
     
 var SurveyDiv = React.createClass({
+    //We want our Survey cards to disappear once submitted, so the getInitialState and removeCard functions provide a boolean
+    //that we check before/while rendering
+    getInitialState: function() {
+        return{
+            showCard: true
+        };
+    },  
+    removeCard: function() {
+        this.setState({showCard: false});
+    },
+    
     render: function() {
-        var itemSurvey = this.props.questions.map(function (itemSurvey) {  
+        //increasing the scope of the props, there has to be a better way to do this.
```

Comment:
@CarlosHL @MCChung303 I bet there's a cleaner way to do this in terms of expanding the scope of these variables, so if someone wants to work on finding the way to do that before me that'd be awesome.